02-running-grakn/01-install-and-run.md

```diff
@@ -70,40 +51,50 @@ Install Grakn Server and Grakn Console:
 # apt install grakn-core-server grakn-core-console
 ```
 
-Having installed Grakn, we can now interact with the [Grakn server](#start-the-grakn-server) and the [Graql console](/docs/running-grakn/console).
+#### Manual Download
+
+Download the [latest release](https://grakn.ai/download?os=linux#core), unzip it in a location on your machine that is easily accessible via terminal.
+
+Having installed or downloaded Grakn, we can now start the [Server](#start-the-grakn-server) and interact with the [Console](/docs/running-grakn/console).
 
 [tab:end]
 
-[tab:Docker]
+[tab:Mac OS X]
 
-Run :
+#### Using Homebrew
 ```
-$ docker run -d -p 48555:48555 graknlabs/grakn
+$ brew tap graknlabs/tap
+$ brew tap-pin graknlabs/tap
+$ brew install grakn-core
 ```
 
+#### Manual Download
+Download the [latest release](https://grakn.ai/download?os=mac_os_x#core), unzip it in a location on your machine that is easily accessible via terminal.
 
-Having started Grakn, we can now interact with the [Grakn server](#start-the-grakn-server) and the [Graql console](/docs/running-grakn/console).
+Having installed or downloaded Grakn, we can now start the [Server](#start-the-grakn-server) and interact with the [Console](/docs/running-grakn/console).
 
 [tab:end]
 
-[tab:Linux]
+[tab:Windows]
 
-### To download the latest release
-Download the [latest release](https://grakn.ai/download?os=linux#core), unzip it in a location on your machine that is easily accessible via terminal.
+#### Manual Download
+Download the [latest release](https://grakn.ai/download?os=windows#core), unzip it in a location on your machine that is easily accessible via command prompt.
 
-Having downloaded Grakn, we can now interact with the [Grakn server](#start-the-grakn-server) and the [Graql console](/docs/running-grakn/console).
+Having installed or downloaded Grakn, we can now start the [Server](#start-the-grakn-server) and interact with the [Console](/docs/running-grakn/console).
 
 [tab:end]
 
-[tab:Windows]
 
-### To download the latest release
-Download the [latest release](https://grakn.ai/download?os=windows#core), unzip it in a location on your machine that is easily accessible via command prompt.
+[tab:Docker]
+
+Run :
+```
+$ docker run -d -p 48555:48555 graknlabs/grakn
+```
 
-Having downloaded Grakn, we can now interact with the [Grakn server](#start-the-grakn-server) and the [Graql console](/docs/running-grakn/console).
+Having installed or downloaded Grakn, we can now start the [Server](#start-the-grakn-server) and interact with the [Console](/docs/running-grakn/console).
 
 [tab:end]
-</div>
```

Comment:
@vmax close the `div`

Reply:
@sorsaffari addressed in https://github.com/graknlabs/docs/pull/106/commits/06ffcb8870ebc3b6fcab899cb8caab5f86256990